dtbootstrap-anima-keyinfra.xml

```diff
@@ -1223,16 +1223,30 @@ locator3  = [O_IPv6_LOCATOR, fe80::1234, 41, nil]]]></artwork>
           detailed in <xref target="RequestVoucherFromMASA"></xref>. As
           detailed in <xref target="securityconsiderations"></xref> the MASA
           might find it necessary to request additional Registrar
-          authentication. Registrars MUST be prepared to support TLS client
-          certificate authentication and HTTP Basic or Digest authentication as
-          described in RFC7030 for EST clients. Implementors are advised that
+          authentication.
+        </t>
+        <t>
+          The Registrar is the client in the BRSKI-MASA connection.  When
+          operating as such, the Registrars MUST be prepared to support TLS client
+          certificate authentication.  The registrar should also be
+          configurable to support HTTP Basic or Digest authentication as
+          described in RFC7030 for EST clients: provision of such credentials
+          of outside the scope of this document and would be part of a supply-chain
+          integration process.
+        </t>
+        <t>
+          Implementors are advised that
           contacting the MASA is to establish a secured REST connection with a
           web service and that there are a number of authentication models
           being explored within the industry. Registrars are RECOMMENDED to
           fail gracefully and generate useful administrative notifications or
           logs in the advent of unexpected HTTP 401 (Unauthorized) responses
           from the MASA.
         </t>
+        <t>
+          In order to support multi-tenant MASA implementations the Registrar
+          SHOULD include the <xref target="RFC6066" /> Server Name Indicator.
+        </t>
```

Comment:
I don't have a technical concern (seems like a good idea) but as an IETF process thing if this is a good idea for all TLS connections used for HTTP/REST interfaces wouldn't there be a good single doc to put the "SHOULD" in? Burying it in every doc that references this stack seems repetitive.